File: pgm/ABC_Java/ABC_115_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final int D = Integer.parseInt (reader.readLine());
        final int twentyThird = 23;
        final int twentyFourth = 24;
        final int twentyFifth = 25;
        final String Christmas = "Christmas";
        final String ChristmasEve = "Christmas Eve";
        final String ChristmasEveEve = "Christmas Eve Eve";
        final String ChristmasEveEveEve = "Christmas Eve Eve Eve";
        System.out.println (D == twentyFifth    ? Christmas
                            : D == twentyFourth ? ChristmasEve
                            : D == twentyThird  ? ChristmasEveEve
                                                : ChristmasEveEveEve);
        return;
    }
}

File: pgm/ABC_R/ABC_115_A.r
con <- file(description = "stdin", open = "r")
arr <- readLines(con = con)
close(con)
one <- 1L
twenty_third <- 23L
twenty_fourth <- 24L
twenty_fifth <- 25L
christmas <- "Christmas"
christmas_eve <- "Christmas Eve"
christmas_eve_eve <- "Christmas Eve Eve"
christmas_eve_eve_eve <- "Christmas Eve Eve Eve"
blank <- ""
space <- " "
D <- strsplit(arr[one], split = space)[[one]][one] |>
    as.integer()
if (D == twenty_fifth) {
    christmas |>
        cat("\n", sep = blank)
} else if (D == twenty_fourth) {
    christmas_eve |>
        cat("\n", sep = blank)
} else if (D == twenty_third) {
    christmas_eve_eve |>
        cat("\n", sep = blank)
} else {
    christmas_eve_eve_eve |>
        cat("\n", sep = blank)
}
q("no")